src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/Cas1PlacementMatchingOutcomesReportTest.kt

```diff
@@ -22,16 +23,18 @@ import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.PlacementRequ
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.PlacementRequestWithdrawalReason
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.PlacementType
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.UserRole
-import uk.gov.justice.digital.hmpps.approvedpremisesapi.reporting.model.Cas1PlacementMatchingOutcomesReportRow
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.toUtcOffsetDateTime
 import java.time.LocalDate
+import java.time.format.DateTimeFormatter
 import java.util.UUID
 
 class Cas1PlacementMatchingOutcomesReportTest : IntegrationTestBase() {
 
   object Constants {
     const val REPORT_MONTH = 1
     const val REPORT_YEAR = 2020
+
+    val DATE_FORMAT: DateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/uuuu")
```

Comment:
interesting to learn about `u` versus `y`: https://stackoverflow.com/a/41178418 Is it worth a note or can we assume that Java/Kotlin devs agree on this as best practice?

Reply:
I've got into the habit of always using uuuu as it helps spot if we accidentally use a negative year (although the likelyhood of that is very slim!). I agree that yyyy is far more understandable, but given we typically always show uuuu next to dd/MM i think it's understandable enough.